src/Client/components/ques/written-ques.tsx

```
@@ -1,143 +1,200 @@
+                  onChange={(e) => handlePoints(question.id, parseInt(e.target.value))}
+                />
+                {!question.isShortAnswer && (
+                  <label>
+                    <Checkbox
+                      checked={question.isShortAnswer}
+                      onChange={(e) => handleShortLongAnswer(question.id, e.target.checked, question.isLongAnswer)}
+                      isDisabled={question.isLongAnswer}
+                    >
+                      Short Answer
+                    </Checkbox>
+                  </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)